app/Http/Controllers/BirdController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\src\Service\saveService;
+use Audubon\Entities\Bird;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\Request;
+use LaravelDoctrine\ORM\Facades\EntityManager;
+
+class BirdController extends Controller
+{
+    public function audubonSociety(): view
+    {
+        /*return view('audubon-society', [
+            'audubon-form' => EntityManager::getRepository(Bird::class)->findAll()
+        ]);*/
+
+        return view('audubon-society');
+    }
+
+    public function saveAudubonSociety(Request $request, saveService $saveService): view
+    {
+        //var_dump($request->input());
+        $saveService->putDataIntoClasses($request->input());
+        return view('redirect-after-submit');
+    }
+
+    public function displayAllSightings(): view
```

Comment:
This name is fine, but often for displaying multiple lists we often will use index as the function name that shows multiple entries of something.  Not every time, but usually.  I probably wouldn't call this out, but it's worth noting.